library/aesce.c

```diff
@@ -17,6 +17,28 @@
  *  limitations under the License.
  */
 
+#if defined(__aarch64__) && !defined(__ARM_FEATURE_CRYPTO) && \
+    defined(__clang__) &&  __clang_major__ < 18 && __clang_major__ > 3
```

Comment:
I'm unhappy about the use of `__clang_major__ < 18`. Clang 15 has been released, and 16 due "soon", but I don't even know when 18 will be released. This seems like a random "some time in the future when we should revisit this" guard - please don't do that. This code should continue to work even if the change hasn't been merged by then, so just remove the `__clang_major__ < 18`. If we get back to it after the merge, then great, and we can put the correct version limit in.

Reply:
Agree. I will change it.